scripts/globals/abyssea/atma.lua

```diff
@@ -281,12 +283,8 @@ local function updateReinfusedMask(player, atmaSlot, atmaValue)
     player:setCharVar("ABYSSEA_LAST_ATMA_INFUSED", newMask)
 end
 
-local function updateHistoryMask(player, atmaValue)
-    local currentAtma = getIdByKeyItemId(atmaValue)
-    local histo =
-    {
-        currentAtma
-    }
+local function getHistoryAtmaArray(player)
+    local histo = {}
```

Comment:
Could this variable be renamed to something a bit more verbose? Something like atmaHistory, etc